packages/react/src/components/UIShell/SideNav.js

```diff
@@ -78,9 +99,9 @@ const SideNav = React.forwardRef(function SideNav(props, ref) {
         {...accessibilityLabel}
         onFocus={event => handleToggle(event, true)}
         onBlur={event => handleToggle(event, false)}
-        onMouseEnter={() => handleToggle(true)}
-        onMouseLeave={() => handleToggle(false)}>
-        {children}
+        onMouseEnter={() => handleToggle({}, true)}
```

Comment:
Any reason not passing along the event?

Reply:
Hi @asudoh good point - as it was returning `true`/`false` before, it should continue to do so. I will change this to `onMouseEnter={() => handleToggle(true, true)}` and `onMouseLeave={() => handleToggle(false, false)}`

Reply:
> as it was returning `true`/`false` before, it should continue to do so.

Right. An interesting part is (as you may have noticed) that the old code didn't have one-arg interface. And other callers of `handleToggle()` passes along the raw event. So I was wondering if we could do:

```suggestion
        onMouseEnter={event => handleToggle(event, true)}
        onMouseLeave={event => handleToggle(event, false)}
```